packages/camera/camera/android/build.gradle

```diff
@@ -27,7 +27,7 @@ project.getTasks().withType(JavaCompile){
 apply plugin: 'com.android.library'
 
 android {
-    compileSdkVersion 29
+    compileSdkVersion 30
```

Comment:
@BeMacized What's the reason for updating it to 30? Is it a requirement for a particular API used in this PR?

Reply:
> @BeMacized What's the reason for updating it to 30? Is it a requirement for a particular API used in this PR?

I'm honestly not quite sure. I think it came from [this commit](https://github.com/flutter/plugins/commit/1d84baea878119e0f5e0443708c19bd6748f9f36). 

cc @mvanbeusekom.

Reply:
@cyanglaz, @bparrishMines, I indeed added this change because the "build_all_plugins_apk" build step returned the following deprecation error (see here for [details](https://cirrus-ci.com/task/5307145929359360?command=main#L293)):

```
/tmp/cirrus-ci-build/packages/camera/camera/android/src/main/java/io/flutter/plugins/camera/DeviceOrientationManager.java:177: warning: [deprecation] getDefaultDisplay() in WindowManager has been deprecated
    int rotation = windowManager.getDefaultDisplay().getRotation();
```

According to the documentation on Android API 30 we should use the `activity.getDisplay()` method instead, which doesn't exists on lower API levels. This is why I changed the `compileSdkVersion` to 30.

Reply:
offline discussion: this is good! LGTM